enterprise/javaee.api/src/org/netbeans/modules/javaee/api/javaee-endorsed-api-6.0.xml

```diff
@@ -20,8 +20,7 @@
     under the License.
 
 -->
-<!DOCTYPE library PUBLIC "-//NetBeans//DTD Library Declaration 1.0//EN" "http://www.netbeans.org/dtds/library-declaration-1_0.dtd">
-<library version="1.0">
+<library version="3.0" xmlns="http://www.netbeans.org/ns/library-declaration/3">
```

Comment:
Validation was failing on the dtd ... and there's a new version of the declaration anyway, so I've updated.